### PR TITLE
fix(DB/Creature): Added patrolling and new spawns to Omgorn the Lost

### PR DESCRIPTION
**Pull request by Asthalor**
## Changes Proposed:
- Changed his spawn timer to 12h
- Added 2 new spawns
- Added routes to the 3 spawns

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6825
- Closes https://github.com/chromiecraft/chromiecraft/issues/992#issuecomment-872203218

## SOURCE:

https://classic.wowhead.com/npc=8201/omgorn-the-lost#comments
https://www.youtube.com/watch?v=Oy8sfxP9zNI

## Tests Performed:

- .go c (51831 or 152280 or 152281)
- Checked his patrolling

## How to Test the Changes:

1.  .go c (51831 or 152280 or 152281)
2. Check his patrolling

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@Helias labels! ^^

**Comment by Asthalor** [comment]
This merge will have bugs, because i added new spawn points without adding them to a pool table, so multiple ormorns can be at the same time.
Please @Helias can you mark it in WIP until i can add the required info? Thanks

File: data/sql/updates/pending_db_world/rev_1627308091046661200.sql [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+-- Added his 3 spawn points to the pool creature and added those 3 to a common pool template (366), with a max of 1 spawn at the same time
+DELETE FROM `pool_template` WHERE `entry` = 366;
+
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`)  VALUES (366, 1, "Omgorn the Lost Spawns");
+
+DELETE FROM `pool_creature` WHERE `guid` IN (51831, 152280, 152281);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (51831, 366, 0, "Omgorn the Lost Dunemaul Compound Spawn");
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (152280, 366, 0, "Omgorn the Lost Eastmoon Ruins Spawn");
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (152281, 366, 0, "Omgorn the Lost Southmoon Ruins Spawn");
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1627308091046661200.sql:22]:
```suggestion
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(51831, 366, 0, "Omgorn the Lost Dunemaul Compound Spawn"),
(152280, 366, 0, "Omgorn the Lost Eastmoon Ruins Spawn"),
(152281, 366, 0, "Omgorn the Lost Southmoon Ruins Spawn");
```

File: data/sql/updates/pending_db_world/rev_1627308091046661200.sql [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+-- Waypoint route Southmoon Ruins (GUID: 152281)
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(1522810,1,-9294.27,-2931.240,9.141,0,0,0,0,100,0),
+(1522810,2,-9282.229,-2922.29,10.919,0,0,0,0,100,0),
+(1522810,3,-9208.737,-2928.27,16.89,0,0,0,0,100,0),
+(1522810,4,-9126.367,-2982.45,40.41,0,0,0,0,100,0),
+(1522810,5,-9126.31,-3046.110,42.49,0,0,0,0,100,0),
+(1522810,6,-9226.81,-3128.54,22.490,0,0,0,0,100,0),
+(1522810,7,-9301.125,-3099.55,11.65,0,0,0,0,100,0),
+(1522810,8,-9307.09,-2989.11,9.249,0,0,0,0,100,0);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1627308091046661200.sql:68]:
```suggestion
-- Waypoint route Dunemaul Compound (GUID: 51831)
INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
(518310,1,-8495.70,-2931.99,8.71,0,0,0,0,100,0),
(518310,2,-8502.15,-2967.93,10.5,0,0,0,0,100,0),
(518310,3,-8462.07,-3062.26,9.79,0,0,0,0,100,0),
(518310,4,-8371.06,-3188.28,9.99,0,0,0,0,100,0),
(518310,5,-8287.775,-3176.50,18.68,0,0,0,0,100,0),
(518310,6,-8189.05,-3044.80,14.31,0,0,0,0,100,0),
(518310,7,-8209.79,-2950.67,17.10,0,0,0,0,100,0),
(518310,8,-8402.98,-2887.02,8.887,0,0,0,0,100,0),
(518310,9,-8431.95,-2890.55,8.62454,0,5000,0,0,100,0),
-- Waypoint route Eastmoon Ruins (GUID: 152280)
(1522800,1,-8971.64,-3539.97,15.39,0,0,0,0,100,0),
(1522800,2,-8963.56,-3546.34,15.100,0,0,0,0,100,0),
(1522800,3,-8813.38,-3539.072,9.411,0,0,0,0,100,0),
(1522800,4,-8810.405,-3367.46,10.92,0,0,0,0,100,0),
(1522800,5,-8828.79,-3337.46,8.735,0,0,0,0,100,0),
(1522800,6,-8850.57,-3337.23,9.390,0,0,0,0,100,0),
(1522800,7,-8905.34,-3401.42,13.47,0,0,0,0,100,0),
(1522800,8, 8990.43,-3435.19,24.91,0,0,0,0,100,0),
-- Waypoint route Southmoon Ruins (GUID: 152281)
(1522810,1,-9294.27,-2931.240,9.141,0,0,0,0,100,0),
(1522810,2,-9282.229,-2922.29,10.919,0,0,0,0,100,0),
(1522810,3,-9208.737,-2928.27,16.89,0,0,0,0,100,0),
(1522810,4,-9126.367,-2982.45,40.41,0,0,0,0,100,0),
(1522810,5,-9126.31,-3046.110,42.49,0,0,0,0,100,0),
(1522810,6,-9226.81,-3128.54,22.490,0,0,0,0,100,0),
(1522810,7,-9301.125,-3099.55,11.65,0,0,0,0,100,0),
(1522810,8,-9307.09,-2989.11,9.249,0,0,0,0,100,0);
```

**Pull request merged**